### ISSUE TITLE
Invalid STIX2 Export

### ISSUE THREAD

**Issue by smackanulty** [OpenCTI-Platform/opencti#571]
Stix2 Validator is showing invalid, and is unable to import file into MISP.

[-] Results for: 2020-03-16T10_06_17.551Z_(ExportFileStix2)_report.json
[X] STIX JSON: Invalid
    [!] Warning: marking-definition--613f2e26-407d-48c7-9eca-b8e91df99dc9: {111} Open vocabulary value 'TLP' should be all lowercase and use hyphens instead of spaces or underscores as word separators.
    [!] Warning: marking-definition--613f2e26-407d-48c7-9eca-b8e91df99dc9: {201} Marking definition `definition_type` should be one of: statement, tlp.
    [!] Warning: report--a12693d2-dc87-4e3c-a708-9da412d87185: {218} labels contains a value not in the report-label-ov vocabulary.
    [X] report--a12693d2-dc87-4e3c-a708-9da412d87185: 'object_refs' is a required property
    [X] observed-data--7112ab66-ad8c-48a2-b316-63a277d44189: 'first_observed' is a required property
    [X] observed-data--7112ab66-ad8c-48a2-b316-63a277d44189: 'last_observed' is a required property
    [X] observed-data--7112ab66-ad8c-48a2-b316-63a277d44189: 'created' is a required property
    [X] observed-data--7112ab66-ad8c-48a2-b316-63a277d44189: 'modified' is a required property
    [X] observed-data--7112ab66-ad8c-48a2-b316-63a277d44189: observed-data--7112ab66-ad8c-48a2-b316-63a277d44189: : Observed Data objects must be in dict format.

Basically the observed-data type is missing the 4 dates and I think the objects dictionary format should be like the below:

 "objects": {
        "0": {
            "type": "x-new-observable",
            "a_property": "foobaz",
            "property_2": 5
        }
    }

Otherwise, thankyou for an interesting product. I look forward to each new release!
